src/js/components/Booking.js

```diff
@@ -11,6 +12,151 @@ class Booking {
 
     thisBooking.render(element);
     thisBooking.initWidgets();
+    thisBooking.getData();
+  }
+
+  getData(){
+    const thisBooking = this;
+
+    const startDateParam = settings.db.dateStartParamKey + '=' + utils.dateToStr(thisBooking.datePicker.minDate);
+    const endDateParam = settings.db.dateEndParamKey + '=' + utils.dateToStr(thisBooking.datePicker.maxDate);
+
+    const params = {
+      booking: [
+        startDateParam,
+        endDateParam,
+      ],
+      eventsCurrent: [
+        settings.db.notRepeatParam,
+        startDateParam,
+        endDateParam,
+      ],
+      eventsRepeat: [
+        settings.db.repeatParam,
+        endDateParam,
+      ],
+    };
+
+    //console.log('getData params', params);
+
+    const urls = {
+      booking:       settings.db.url + '/' + settings.db.bookings
+                                     + '?' + params.booking.join('&'),
```

Comment:
Bardziej czytelne i bardziej zwięzłe
```suggestion
      booking:       `${settings.db.url}/${settings.db.bookings}?${params.booking.join('&')}`,
```